### ISSUE TITLE
Add default Outlook category color

### ISSUE THREAD

**Issue by joslange** [Puzzlepart/did#437]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Standardize Outlook calendar category colors.

### Description
To standardize color coding in employee calendars and mark different projects accordingly, there should be a way to select a default category color on each project. If possible, use the same color for all customer projects and different colors for different types of internal projects (pzl, iam, pmi, etc)

For those that use color coding today, adding projects from DID with random colors will mess everything up and needs to be modified manually. Also, with standardized color coding it will be easier to view other peoples calendars if the same color coding is used. 

**We can do one of the following**
1. Pick one of the presets colors, and always use that one. 
2. Let the user pick the color when creating the category from Did
3. Add a user setting to decide what preset color to use for outlook categories
4. Set the preset color on the project entity
5. Set the preset color on the customer entity
